src/commands/ContainerAppOverwriteConfirmStep.ts

```diff
@@ -14,6 +14,8 @@ export class ContainerAppOverwriteConfirmStep<T extends IContainerAppContext> ex
     public hideStepCount: boolean = true;
 
     public async prompt(context: T): Promise<void> {
+        context.telemetry.properties.hasUnsupportedFeatures = 'true';
```

Comment:
That's easy enough, like you said it's just a bit uglier, but I can probably just go in and log non-default states.  The initialization could work, but I think it might end up being slightly more complicated, so I'll prefer the other way for now unless it ends up looking bad